keel-core/src/test/kotlin/com/netflix/spinnaker/keel/serialization/BooleanPropertySerializationTests.kt

```diff
@@ -0,0 +1,34 @@
+package com.netflix.spinnaker.keel.serialization
+
+import com.fasterxml.jackson.databind.node.ObjectNode
+import com.netflix.spinnaker.keel.core.api.ActionMetadata
+import com.netflix.spinnaker.keel.core.api.ArtifactSummaryInEnvironment
+import dev.minutest.junit.JUnit5Minutests
+import dev.minutest.rootContext
+import java.time.Instant.now
+import strikt.api.expectThat
+import strikt.jackson.at
+import strikt.jackson.has
+import strikt.jackson.isObject
+import strikt.jackson.path
+
+class BooleanPropertySerializationTests : JUnit5Minutests {
```

Comment:
Should this be renamed to something else? `Boolean` with no boolean is a bit confusing :)

Reply:
Yes, I'll do that. Good point.